### PR TITLE
[Javascript] ESLint --fix autofixed files

### PR DESCRIPTION
**Pull request by Aliandi**
### Description
Files with warnings thrown by the current ESLint rules are fixed here. 

### Changes made
The files were affected by the ESLint `--fix` option removing the warnings.
image
### Testing
Build and tests are running well.
image

image

### CODE REVIEW

File: JavaScript/packages/recognizers-choice/src/choice/extractors.ts [hunk 1]
```diff
@@ -65,7 +65,8 @@ export class ChoiceExtractor implements IExtractor {
             let topResult = partialResults.reduce((top, value) => top = top.data.score < value.data.score ? value : top, partialResults[0]);
             topResult.data.otherMatches = partialResults.filter(r => r !== topResult);
             results.push(topResult);
-        } else {
+        }
+ else {
```
Comment (STayinloves) [inline JavaScript/packages/recognizers-choice/src/choice/extractors.ts]: auto-fix failure.

File: JavaScript/packages/recognizers-choice/src/choice/extractors.ts [hunk 2]
```diff
@@ -111,9 +112,11 @@ export class ChoiceExtractor implements IExtractor {
                     tokens.push(token);
                     token = '';
                 }
-            } else if (!(this.config.tokenRegex.test(c) || StringUtility.isWhitespace(c))) {
+            }
+ else if (!(this.config.tokenRegex.test(c) || StringUtility.isWhitespace(c))) {
```
Comment (STayinloves) [inline JavaScript/packages/recognizers-choice/src/choice/extractors.ts]: same

**Comment by Aliandi** [comment]
Yes @tellarin, we had some issues while rebasing with master. We will be fixing those ASAP.

**Comment by tellarin** [comment]
@Konigstiger @Aliandi It seems there are still merge issues.

**Comment by Aliandi** [comment]
@tellarin We ended up creating a new PR (#1767) from a new branch as we were having plenty of build issues with this one. Sorry for the delay!

**Pull request closed without merging**